src/main/java/org/trigon/commands/GearRatioCalculationCommand.java

```diff
@@ -51,6 +66,24 @@ public GearRatioCalculationCommand(TalonFXMotor motor, CANcoderEncoder encoder,
         );
     }
 
+    /**
+     * Creates a new GearRatioCalculationCommand.
+     * This constructor will record the starting positions without any delay. This might be problematic when the system has backlash.
+     * When you have backlash and want to account for it with a measuring delay, use the other constructor.
+     *
+     * @param rotorPositionSupplier   a supplier that returns the current position of the rotor in degrees
+     * @param encoderPositionSupplier a supplier that returns the current position of the encoder in degrees
```

Comment:
the unit doesn't matter
also on lines 90-91 
the only thing that matters is that the encoder supplier and rotor supplier have the same unit